### PR TITLE
Fixed bug that failed to include certain abstract classes in schema

### PR DESCRIPTION
**Pull request by pmantica1**

### CODE REVIEW

File: graphql_compiler/schema_generation/graphql_schema.py [hunk 1]
```diff
@@ -320,18 +320,21 @@ def get_graphql_schema_from_schema_graph(schema_graph, class_to_field_type_overr
     for non_graph_cls_name in schema_graph.non_graph_class_names:
         if non_graph_cls_name in hidden_classes:
             continue
+        if not schema_graph.get_element_by_class_name(non_graph_cls_name).abstract:
```
Comment (obi1kenobi) [inline graphql_compiler/schema_generation/graphql_schema.py]: Please add an explanation for why we're continuing here. The `hidden_classes` part is obvious, this is less so.

File: graphql_compiler/schema_generation/graphql_schema.py [hunk 2]
```diff
@@ -320,18 +320,21 @@ def get_graphql_schema_from_schema_graph(schema_graph, class_to_field_type_overr
...
+            continue
 
         cls_subclasses = schema_graph.get_subclass_set(non_graph_cls_name)
         # No need to add the possible abstract class if it doesn't have subclasses besides itself.
         if len(cls_subclasses) > 1:
```
Comment (bojanserafimov) [inline graphql_compiler/schema_generation/graphql_schema.py]: is this if statement necessary?

Comment (bojanserafimov) [reply graphql_compiler/schema_generation/graphql_schema.py]: resolved offline

File: graphql_compiler/schema_generation/graphql_schema.py (hunk 1 above)
Comment (pmantica1) [reply graphql_compiler/schema_generation/graphql_schema.py]: There is a comment explaining that the class has to be  abstract at the top of the block.

**Comment by pmantica1** [comment]
I'll add a reproducing test sometime tonight. Trying to change the schema for the autogenerated one right now.

**Pull request closed without merging**